CONTRIBUTING.md

```diff
@@ -1,15 +1,7 @@
 # Contributing
 
-## CHANGELOG 
-
-Authors are expected to include a changelog file with their pull requests. The changelog file
-should be a new file created in the `changelogs/unreleased` folder. The file should follow the
-naming convention of `pr-username` and the contents of the file should be your text for the
-changelog.
-
-    velero-plugin-for-vsphere/changelogs/unreleased   <- folder
-        000-username            <- file
-
+## Opening PRs
+When opening a pull request, please fill out the checklist supplied the template. This will help others properly categorize and review your pull request.
```

Comment:
Do we want to explicitly emphasize that release-note should be filled in?

Reply:
Added.